README.md

```diff
@@ -0,0 +1,177 @@
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <h3><b>Trial Twor</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
```

Comment:
- [ ] I'll suggest you kindly remove the links for `Live Demo`, `Run tests`, `Deployment`, and `FAQs` from your readme file if you are not planning to add their respective sections because now, these links are not working which is making your readme file a bit unprofessional. Doing so will make your readme file more professional and will give a better user experience. 😺 😄

Reply:
- [ ] Kindly implement these suggestions of the previous reviewer to make your readme looks more professional. 😊